### ISSUE TITLE
Consensus Spectrum Intensities incorrect

### ISSUE THREAD

**Issue by PMSeitzer** [eugenemel/maven#213]
To reproduce:
Load `Solve06_NegMeth01_02_L0.mzML`
View MS2 scans associated with `LPE(18:1/0:0)_IS[M-H]-` (precursor m/z = 485.33786011)
fragment at m/z = 140.0118 does not produce the correct average (testable in R).

**Comment by PMSeitzer** [comment]
related `mass_spec` issue:
https://github.com/calico/mass_spec/issues/381

**Comment by PMSeitzer** [comment]
see `20200505-NIST1950-neg-mode.R`:

```
> results_is_x<- results_is_all_ms2_intensities_compounds %>% dplyr::filter(grepl("ethanolamine-P",fragmentLabel)) %>% dplyr::select(sample, ref_ms1_mz, ref_ms2_mz, intensity, ms2_intensity) %>% dplyr::filter(sample=="Solv06_NegMeth01_02_L0.mzML") %>% dplyr::filter(!is.na(intensity))
> results_is_x
```
produces
```
                      sample ref_ms1_mz ref_ms2_mz intensity ms2_intensity
1 Solv06_NegMeth01_02_L0.mzML   485.3379   140.0118  5509.830      795423.1
2 Solv06_NegMeth01_02_L0.mzML   485.3379   140.0118 12166.201      795423.1
3 Solv06_NegMeth01_02_L0.mzML   485.3379   140.0118  4364.986      795423.1
4 Solv06_NegMeth01_02_L0.mzML   485.3379   140.0118  3056.969      795423.1
5 Solv06_NegMeth01_02_L0.mzML   485.3379   140.0118  7570.414      795423.1
```

The correct intensity is 
```
> mean(results_is_x$intensity)
[1] 6533.68
```

The intensity that is being displayed in the plot, however, is
```
> (sum(results_is_x$intensity) + 12166.201)/6
[1] 7472.434
```
So the extra scan is being added twice.

**Comment by PMSeitzer** [comment]
Fixed as of 23d01162561763854c44eeba7a9662a1a92262c2
maven_core commit:
https://github.com/eugenemel/maven_core/commit/7029994f4354d4c02900ecc57d0ef19271bc783f
